README.md

```diff
@@ -46,10 +46,11 @@ Building Timewarrior yourself requires
 
 There are two ways to retrieve the Timewarrior sources:
 
-* Clone the repository from GitHub,
+* Clone the repository from GitHub and update required submodules,
   ```
   git clone --recurse-submodules https://github.com/GothenburgBitFactory/timewarrior
   cd timewarrior
+  git submodule update --init --recursive
```

Comment:
I would remove this unnecessary command.

Reply:
Apologies, I thought my commit got updated with my last push and I had identical modifications for this PR in mind.